image/image.go

```diff
@@ -65,7 +65,14 @@ func (i *genericImage) Manifest() ([]byte, string, error) {
 			return nil, "", err
 		}
 		i.cachedManifest = m
-		if mt == "" {
+		is_reqmt := false
+		for _, reqmt := range i.requestedManifestMIMETypes {
+			if mt == reqmt {
+				is_reqmt = true
+				break
+			}
+		}
+		if !is_reqmt {
```

Comment:
(… with another look, my thesis that `Manifest` should return the same MIME type as `GetManifest` immediately fails per the already existing guessing code.

I guess we’ll have to figure out the proper place after `skopeo copy` learns more about format conversions.)

`text/plain` is so clearly invalid that
```go
if mt == "" || mt == "text/plain" { // Crane can return "text/plain" $more_description
```
seems like a nice surgical fix, if it is sufficient that is.

Let’s wait for @runcom though.